lib/richmond/logging.rb

```diff
@@ -1,8 +1,13 @@
 require 'logger'
 
 module Richmond
+
   module Logging
 
+    def note message
+      logger.info message
+    end
+
```

Comment:
Rather than force people to do this:

```ruby
logger.info 'something'
```

Why not isolate the logger from users and just expose the methods that they'll want to use?

Like in this case, I just threw in a method called ```note``` because that's all I felt it was doing.

It's more enterprise-y to use Logger.info... but I'd craft the code of the gem to fit the usage within the gem.  I hate details like ```Logger``` bleeding all over my gems.